src/plugins/path/path.c

```diff
@@ -9,9 +9,33 @@
 #include "path.h"
 
 #ifndef HAVE_KDBCONFIG
+
 #include "kdbconfig.h"
+
 #endif
 
+// Used to delete last colon in the error message
```

Comment:
It would be very nice, if you could add [Doxygen](http://doxygen.nl) comments for all functions.